module/Activity/src/Activity/Mapper/Activity.php

```diff
@@ -113,6 +113,8 @@ public function getUpcomingActivitiesForMember($user)
         foreach ($user->getMember()->getCurrentOrganInstallations() as $organ) {
             $result = array_merge($result, $this->getUpcomingActivitiesByOrgan($organ));
         }
+        
+        $result = array_unique($result);
```

Comment:
Unfortunately, this will throw a fatal _"Object of class Activity\Model\Activity could not be converted to string"_ error ([ref](http://php.net/manual/en/function.array-unique.php))

Reply:
I have resolved the issue, but haven't tested it yet.

Reply:
Tested as well now